docs/Files Needed for 64 bit Support.md

```diff
@@ -0,0 +1,65 @@
+## Files Needed to Build with LabVIEW 64-bit
+Because VeriStand 2021 and later only support LabVIEW 64-bit, several directories of VeriStand LabVIEW support files need duplicated to the LabVIEW 32-bit directory to build this custom device for those versions. This document details which directories to copy, as well as which driver support files need exposed to LabVIEW 64-bit.
```

Comment:
```suggestion
Because VeriStand 2021 and later only support LabVIEW 64-bit, several directories of VeriStand LabVIEW support files need to be duplicated to the LabVIEW 32-bit directory to build this custom device for those versions. This document details which directories to copy, as well as which driver support files need to be exposed to LabVIEW 64-bit.
```